packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/azure/appatidp.md

```diff
@@ -0,0 +1,23 @@
+1. Create an Azure app using these [instructions](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
+
+2. Skip the **Redirect URI** section for now. We come back to it later.
+
+3. Save the Application ID value so that you can add it to the Okta configuration in the <GuideLink link="../configure-idp-in-okta">next section</GuideLink>.
```

Comment:
Are we saving this here, or copying it down to paste it in later? Change "Save" to "Copy"?